src/ng-select/ng-select.component.html

```diff
@@ -53,7 +53,7 @@
              [class.marked]="item === itemsList.markedItem">
 
             <ng-template #defaultOptionTemplate>
-                <span class="ng-option-label" [innerHTML]="item.label"></span>
+                <span class="ng-option-label" [innerHTML]="item.highlightedLabel"></span>
```

Comment:
This approach is limited to default filter only and will not work if using custom [typeahead]. It would be better to create directive [ngOptionHighlight]="filterValue" which would be responsible for highlighting label. 

```html
<span [ngOptionHighlight]="filterValue" class="ng-option-label" [innerHTML]="item.label"></span>
```

Also users will be able to apply this directive even if they are using custom option templates.

Reply:
Yep good idea, done.